conda_smithy/templates/azure-pipelines-osx.yml.tmpl

```diff
@@ -0,0 +1,83 @@
+jobs:
+- job: osx_64
+  pool:
+    vmImage: xcode9-macos10.13
+  strategy:
+    maxParallel: 8
+    matrix:
+    {%- for config, platform in configs | sort %}
+    {%- if platform.startswith('osx') %}
+      {{ config }}:
+        CONFIG: {{ config}}
+    {%- endif %}
+    {%- endfor %}
+
+  steps:
+  # TODO: Fast finish on azure pipelines?
+  - script: |
+      echo "Fast Finish"
+      {{ fast_finish }}
```

Comment:
Unfortunately, Azure Pipelines doesn't yet support fast finish of jobs created by a matrix.

Reply:
Ah so this actually does something different. Though we probably need to fix it as well. We have [our own fast finish script for CIs]( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/master/recipe/ff_ci_pr_build.py ) (particularly if the provider doesn't support it or the provider implementation fails).

That said, it looks like Marius has made this an empty string for now. So it isn't running the script at all, which is fine to start. We can update this once the CI fast finish script handles the Azure case. Raised as issue ( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/32 ).